src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public static final int DATE = 0;
+    public static final int EMPLOYEE_NAME = 1;
+    public static final int EMPLOYEE_WORKED_HOURS = 2;
+    public static final int EMPLOYEE_INCOME = 3;
+    public static final String DATE_FORMAT = "dd.MM.yyyy";
+
+    public String getSalaryInfo(String[] names, String[] data,
+                                String dateFrom, String dateTo) {
+        StringBuilder reportBuilder = new StringBuilder();
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
```

Comment:
[If you create a formatter, make it a constant field.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field)